### PR TITLE
Misc Restoration Cleanup

### PR DESCRIPTION
**Pull request by NickPhura**
Misc tweaks/cleanup

# Overview

## This PR contains the following changes

- Cleaned up the keycloak/sso env vars
- Removed some unused code
- Fixed a few code smells
- Added the keycloak service from SIMS.
  - Supports the GCNotify service, which isn't used, but probably will be in the near future Id assume.

## This PR contains the following types of changes

- [ ] New feature (change which adds functionality)
- [ ] Enhancement (improvements to existing functionality)
- [ ] Bug fix (change which fixes an issue)
- [ ] Misc cleanup / Refactoring / Documentation
- [ ] Version change

## Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### General

- [ ] I have performed a self-review of my own code

### Code

- [ ] New files/classes/functions have appropriately descriptive names and comment blocks to describe their use/behaviour
- [ ] I have avoided duplicating code when possible, moving re-usable pieces into functions
- [ ] I have avoided hard-coding values where possible and moved any re-usable constants to a constants file
- [ ] My code is as flat as possible (avoids deeply nested if/else blocks, promise chains, etc)
- [ ] My code changes account for null/undefined values and handle errors appropriately
- [ ] My code uses types/interfaces to help describe values/parameters/etc, help ensure type safety, and improve readability

### Style

- [ ] My code follows the established style conventions
- [ ] My code uses native material-ui components/icons/conventions when possible

### Documentation

- [ ] I have commented my code sufficiently, such that an unfamiliar developer could understand my code
- [ ] I have added/updated README's and related documentation, as needed

### Tests

- [ ] I have added/updated unit tests for any code I've added/updated
- [ ] I have added/updated the Postman requests/tests to account for any API endpoints I've added/updated

### Linting/Formatting

- [ ] I have run the linter and fixed any issues, as needed  
       _See the `lint` commands in package.json_
- [ ] I have run the formatter and fixed any issues, as needed  
       _See the `format` commands in package.json_

### SonarCloud

- [ ] I have addressed all SonarCloud Bugs, Vulnerabilities, Security Hotspots, and Code Smells

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Screenshots

Please add any relevant UI screenshots, if applicable.

### CODE REVIEW

File: app/src/components/map/components/DrawControls.tsx [hunk 1]
```diff
@@ -134,7 +134,7 @@ const DrawControls: React.FC<IDrawControlsProps> = (props) => {
 
           return new L.Marker([latlng.lat, latlng.lng]);
         },
-        onEachFeature: function (feature, layer) {
+        onEachFeature: function (_feature, layer) {
```
Comment (NickPhura) [inline app/src/components/map/components/DrawControls.tsx:137]: Apparently sonar prefers unused variables to start with an underscore, at least more recently.  Maybe their rules got updated.

File: api/src/database/db.test.ts [hunk 2]
```diff
@@ -1,12 +1,20 @@
 import { expect } from 'chai';
 import { describe } from 'mocha';
 import * as pg from 'pg';
-import Sinon from 'sinon';
+import Sinon, { SinonStub } from 'sinon';
+import SQL from 'sql-template-strings';
 import { SYSTEM_IDENTITY_SOURCE } from '../constants/database';
 import { HTTPError } from '../errors/custom-error';
 import { setSystemUserContextSQL } from '../queries/database/user-context-queries';
 import * as db from './db';
```
Comment (curtisupshall) [inline api/src/database/db.test.ts:9]: Can we delete this line?

**Review by curtisupshall — APPROVED** [review]

File: api/src/database/db.test.ts (hunk 2 above)
Comment (NickPhura) [reply api/src/database/db.test.ts:9]: Its still used in a stub further down

Comment (NickPhura) [reply api/src/database/db.test.ts:9]: In order to stub out its content we have to import it this way.

**Pull request merged**